lib/companies/pending_changes.ex

```diff
@@ -138,9 +142,17 @@ defmodule Companies.PendingChanges do
     error
   end
 
+  defp apply_changes("create", changeset), do: Repo.insert(changeset)
+  defp apply_changes("update", changeset), do: Repo.update(changeset)
+  defp apply_changes("delete", record), do: Repo.delete(record)
+
   defp resource_module("company"), do: Company
   defp resource_module("industry"), do: Industry
   defp resource_module("job"), do: Job
+  defp resource_module(_), do: {:error, :module_not_recognized}
+
+  defp drop_relations(job = %Job{}), do: Map.drop(job, [:company])
```

Comment:
I don't think we need these methods potentially but if we do, they can probably be consolidated into one.  `Map.drop/2` can handle keys that don't exist.